content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+| `servicegraph.service.externalPort` | `8088` |  | `` |
+| `servicegraph.service.internalPort` | `8088` |  | `` |
+| `servicegraph.ingress` | `servicegraph.local` |  | `` |
+| `servicegraph.ingress.enabled` | `false` |  | `` |
+| `servicegraph.ingress.hosts` | `servicegraph.local` |  | `` |
+| `servicegraph.ingress.annotations` | `` |  | `` |
+| `servicegraph.ingress.tls` | `` |  | `` |
+| `servicegraph.resources` | `{}` |  | `` |
+| `servicegraph.prometheusAddr` | `http://prometheus:9090` |  | `` |
+| `tracing` | `zipkin.local` |  | `` |
+| `tracing.enabled` | `false` |  | `` |
+| `tracing.jaeger` | `` |  | `` |
+| `tracing.jaeger.enabled` | `false` |  | `` |
+| `tracing.jaeger.memory` | `` |  | `` |
+| `tracing.jaeger.memory.max_traces` | `50000` |  | `` |
+| `tracing.replicaCount` | `1` |  | `` |
+| `tracing.image` | `` |  | `` |
+| `tracing.image.repository` | `jaegertracing/all-in-one` |  | `` |
+| `tracing.image.tag` | `1.5` |  | `` |
+| `tracing.service` | `` |  | `` |
+| `tracing.service.name` | `http` |  | `` |
+| `tracing.service.type` | `ClusterIP` |  | `` |
+| `tracing.service.externalPort` | `9411` |  | `` |
+| `tracing.service.internalPort` | `9411` |  | `` |
+| `tracing.service.uiPort` | `16686` |  | `` |
+| `tracing.ingress` | `zipkin.local` |  | `` |
+| `tracing.ingress.enabled` | `false` |  | `` |
+| `tracing.ingress.hosts` | `zipkin.local` |  | `` |
+| `tracing.ingress.annotations` | `` |  | `` |
+| `tracing.ingress.tls` | `` |  | `` |
+| `tracing.resources` | `{}` |  | `` |
+| `kiali` | `` |  | `` |
+| `kiali.enabled` | `false` |  | `` |
+| `kiali.replicaCount` | `1` |  | `` |
+| `kiali.image` | `` |  | `` |
+| `kiali.image.repository` | `kiali/kiali` |  | `` |
+| `kiali.image.tag` | `0.3.1.Alpha` |  | `` |
+| `kiali.ingress` | `` |  | `` |
+| `kiali.ingress.enabled` | `false` |  | `` |
+| `kiali.ingress.annotations` | `` |  | `` |
+| `kiali.ingress.tls` | `` |  | `` |
+| `kiali.resources` | `{}` |  | `` |
+| `kiali.dashboard` | `` |  | `` |
+| `kiali.dashboard.username` | `admin` |  | `` |
+| `kiali.dashboard.password` | `admin` |  | `` |
+<!-- AUTO-GENERATED-END -->
+
+The Helm chart also offers significant customization options per individual
+service. Customize these per-service options at your own risk. The per-service options are exposed via
+the [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) file.
+
+## Customization example: traffic management minimal set
+
+Istio is equipped with a rich and powerful set of features and some users may need only subset of those. For instance, users might be interested only
+in installing the minimal set required to Istio's traffic management.
+Helm customization provides the option to install a subset by enabling those features of interest and disabling the ones that aren't required.
+
+In this example we will install Istio with only a minimal set of components necessary to conduct [traffic management](/docs/tasks/traffic-management/).
```

Comment:
Generally, avoid using "we". Just say

This example shows how to install the minimal set of components necessary to use traffic management features